### ISSUE TITLE
Multiple number widget

### ISSUE THREAD

**Issue by lukabirsa** [metabase/metabase#5234]
As a user, I would like to display multiple numbers in a single number widget.

An example would be: I take my orders data and create two aggregations. One is quantity, the other value. Right now I need to use a table to display this data. #5233

**Comment by mmuschalik** [comment]
I agree, I would love to put several numbers together in a single widget. Example:
<#Total Succeeded> / <#Total Failed>

**Comment by geofftang** [comment]
Having two numbers would be useful for being able to format % to be a less prominent number. It's clunkier with two numbers necessary

<img width="238" alt="Image" src="https://github.com/user-attachments/assets/d094322a-b539-4f25-a686-b1f74c81ed90" />
